src/wwwroot/controllers/MainCtrl.js

```diff
@@ -33,8 +33,8 @@
     var loaderFreeTrial = function () {
       if (auth.getUserName() != null) {
         auth.getLimitsByAccount().then(function(limits) {
-          if (limits.freeTrialEndDate) {
-            UpdateTrialHeader(moment(limits.freeTrialEndDate).toDate());
+          if (limits.data.freeTrialEndDate) {
+            UpdateTrialHeader(moment(limits.data.freeTrialEndDate).toDate());
```

Comment:
Maybe, the responsibility of _unwrapping_ limits data is part of `auth.getLimitsByAccount()` function in place of `loaderFreeTrial()`